cmd/build.go

```diff
@@ -4,7 +4,7 @@ import (
 	"encoding/base64"
 	"errors"
 	"fmt"
-	v1 "github.com/onepanelio/core/pkg"
+	"github.com/onepanelio/cli/cloud/storage"
```

Comment:
Why this change?

Reply:
@rushtehrani I'm working on removing the dependency on onepanel-core. Without this change, I'd have to update onepanel-core with these changes and pull the updated version. I think I just need the ptr utils and that's it.

Reply:
@rushtehrani Actually, looks like that was it. No further need for the dependency.

Reply:
got it, we'd have to eventually pull these into a shared package.

Reply:
@rushtehrani Agreed.